.gitignore

```diff
@@ -12,6 +12,9 @@ addon
 *~
 robots.txt
 
+#ignore local config
+/config/local.ini.php
```

Comment:
The `doc/html` below has no leading `/` I guess one is incorrect.

Reply:
It doesn't really matter, it just means that all folders named `html` under any folder named `doc` will be ignored, whatever the level in the file tree.

Reply:
Ah, ok. But the favicon file and addon directory should be ignored from the repository, don't they? The one is a node specific file and the other has its own repo.